### ISSUE TITLE
Why is RTC_3V connected to 3.3V?

### ISSUE THREAD

**Issue by cdwilson** [sparkfun/MicroMod_Processor-RP2040#1]
The RP2040 MicroMod shorts RTC_3V directly to 3.3V. What's the reason why these are shorted together on this processor board?

![image](https://user-images.githubusercontent.com/14631/111022237-68ada980-8386-11eb-9b0e-43276a5f098d.png)

When 3.3V_EN is disabled on a carrier board, won't this cause the RTC battery to attempt to backfeed any devices on 3.3V rail (including devices on the carrier board 3.3V)?

The MicroMod specs for RTC_3V say:
> 3V provided by external coin cell or mini battery. Max draw=100μA. Connect to pins maintaining an RTC during power loss. Can be left NC.

The 3V3_LED on the carrier board by itself will exceed 100μA at 3V. Doesn't this violate the above spec for RTC_3V?

**Comment by nseidle** [comment]
Yep, it's a mistake. I carried it over from Artemis (including the blocking diode). Then, after discussions with RPi it became obvious the RP2040 really doesn't have low power modes. I removed the diode but failed to remove the 3.3V RTC connection. While this connection will put a load on an external RTC battery I don't foresee any long term issues. You would see the same behavior with an unplugged Artemis or other micro that is left in full power mode.

I'll leave this open incase we rev the board in the future.
